lib/ProductOpener/DataQualityFood.pm

```diff
@@ -740,6 +740,10 @@ sub check_nutrition_data ($product_ref) {
 						$nid_non_zero++;
 					}
 				}
+				# negative value in nutrition table
+				if ($product_ref->{nutriments}{$nid} < 0) {
+					push @{$product_ref->{data_quality_errors_tags}}, "en:nutrition-value-negative-$nid2";
```

Comment:
cool :-) 

I think we should also add a generic tag: en:nutrition-value-negative

So you have the generic tag + the more precise one (so we can see all product with  a negative nutrition value).

Reply:
We have a taxonomy for data_quality, so we could use it for generic tags.

Reply:
Thanks for the inputs.

Would that be fine to leave DataQualityFood.pm as it is now and to add all the errors in taxonomies/data_quality.txt (en:nutrition-value-negative, and all children: en:nutrition-value-negative-salt, en:nutrition-value-negative-carbohydrates, etc.)?

Reply:
> Would that be fine to leave DataQualityFood.pm as it is now and to add all the errors in taxonomies/data_quality.txt (en:nutrition-value-negative, and all children: en:nutrition-value-negative-salt, en:nutrition-value-negative-carbohydrates, etc.)?

That sounds good to me.